SalesforceCommerceCloud/pwa-kit
# Edit URL customization copy

# Description



# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Tweak language and style for clarity and to match Salesforce style
- Update docs links

# How to Test-Drive This PR

- Review the copy changes for technical accuracy
- Double-check the new docs URLs

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
